### PR TITLE
FE for aggregates

### PR DESCRIPTION
**Pull request by hnykda**
Touches #192 , FE part.

### CODE REVIEW

**Comment by hnykda** [comment]
Just a placeholder, seems fairly easy once we have data.

File: src/server/templates/lines.html [hunk 1]
```diff
@@ -8,7 +8,6 @@ <h5>Active Infections <span id="infections-date"></span></h5>
       <thead>
         <th>Confirmed</th>
         <th>Estimated</th>
-        <!--<th>90% CI</th>-->
```
Comment (gavento) [inline src/server/templates/lines.html]: Why not leave it? We may reactivate it later.

File: src/server/templates/model.html [hunk 2]
```diff
@@ -53,6 +53,14 @@
...
       {% include "lines.html" %}
 
+      <p>
+        At the peak of the epidemic, under this scenario, there will be
+        requiring <span id="selected-region-beds"></span> hospital beds and
+        <span id="selected-region-active-infections"></span> active infections,
+        <span id="selected-region-icu-units"></span> ICU units.
+        Overall there will be <span id="selected-region-infections"></span> infections,
+        which can lead do <span id="selected-region-deaths"></span> deaths.
+      </p>
```
Comment (gavento) [inline src/server/templates/model.html]: I would prefer a list (with commenting paragraph), e.g.:

Under this scenario, with high confidence (90%) we estimate:
* The **maximum number of simultaneously infected people** will be **X-Y** (A-B%)
* The **total number of people infected** will be **X-Y** (A-B%)
* St. similar for beds a daths?

**Comment by gavento** [comment]
The data is already deployed at
https://storage.googleapis.com/static-covid/static/data-main-v3.json (new
key in `data` in every country, note it is per 1000 pops)

File: src/server/templates/model.html (hunk 2 above)
Comment (hnykda) [reply src/server/templates/model.html]: That sounds better. Added.

**Comment by hnykda** [comment]
This is how it looks now:

image

File: src/server/static/js/lines.js [hunk 3]
```diff
@@ -63,6 +63,35 @@ function updateInfectionTotals() {
   );
   */
   d3.select("#infections-population").html(formatInfectionTotal(population));
+
+  var mitigation = getMitigationId();
+  console.log(mitigation, data.mitigation_stats, data.mitigation_stats["None"]);
+  // TODO: this line below should be selected dynamically
```
Comment (hnykda) [inline src/server/static/js/lines.js]: help needed

**Comment by hnykda** [comment]
image
variant with `K` and `M` for big values

**Comment by jk3141** [comment]
Nice

**Comment by hnykda** [comment]
OK, I am going to merge and deploy it, but would be nice if someone was able to solve that todo @jeromezng @wolverdude :pray: ... I _think_ it may because the selection of the mitigation happens after the code for getMitigationId is run? :man_shrugging:

**Pull request merged**